testutil/src/test/java/org/spine3/test/NullToleranceTestBuilderShould.java

```diff
@@ -0,0 +1,150 @@
+        NullToleranceTest.newBuilder()
+                         .addDefaultValue(null);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void throw_exception_when_pass_empty_string_as_excluded_method_name(){
+        NullToleranceTest.newBuilder()
+                         .excludeMethod("");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void throw_exception_when_pass_invalid_excluded_method_name(){
+        NullToleranceTest.newBuilder()
+                         .excludeMethod("incorrect method name");
+    }
+
+    @Test
+    public void return_target_class() {
+        final Class<?> targetClass = NullToleranceTest.newBuilder()
+                                                      .setClass(NullToleranceTestBuilderShould.class)
+                                                      .getTargetClass();
+        assertEquals(NullToleranceTestBuilderShould.class, targetClass);
+    }
+
+    @Test
+    public void return_default_values_map() {
+        final NullToleranceTest.Builder builder = NullToleranceTest.newBuilder();
+
+        int expectedSize = 0;
+        int actualSize = builder.getDefaultValues()
+                                .size();
+        assertEquals(expectedSize, actualSize);
+
+        builder.addDefaultValue(0);
+        builder.addDefaultValue('\u000f');
+        expectedSize = 2;
+        actualSize = builder.getDefaultValues()
+                            .size();
+        assertEquals(expectedSize, actualSize);
+
+        final Map<? super Class, ? super Object> expectedMap = newHashMap();
+        expectedMap.put(Integer.class, 0);
+        expectedMap.put(Character.class, '\u000f');
+        assertEquals(expectedMap, builder.getDefaultValues());
+    }
+
+    @Test
+    public void return_excluded_method_names_set() {
+        final NullToleranceTest.Builder builder = NullToleranceTest.newBuilder();
+
+        int expectedSize = 0;
+        int actualSize = builder.getExcludedMethods()
+                                .size();
+        assertEquals(expectedSize, actualSize);
+
+        final String firstMethodName = "method";
+        final String secondMethodName = "secondMethod";
+        builder.excludeMethod(firstMethodName);
+        builder.excludeMethod(secondMethodName);
+        expectedSize = 2;
+        actualSize = builder.getExcludedMethods()
+                            .size();
+        assertEquals(expectedSize, actualSize);
+
+        final Set<String> expectedMethodNames = newHashSet(firstMethodName, secondMethodName);
+        final Set<String> actualMethodNames = builder.getExcludedMethods();
+        assertEquals(expectedMethodNames, actualMethodNames);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void do_not_build_null_tolerance_instance_without_target_class() {
+        NullToleranceTest.newBuilder()
+                         .build();
+    }
+
+    @Test
+    public void build_null_tolerance_class_instance() {
```

Comment:
`null_tolerance_instance` is clear already.
This method is more like
`build_instance_when_all_required_fields_are_set`.